app/views/policy_classes/_comment_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= content_tag(
```

Comment:
Not using `fields_for` or govuk form builder helpers, since this partial needs to be refreshed without the context of the whole form.